sagemaker-inference-recommender/auto-scaling/optimize_endpoint_scaling.ipynb

```diff
@@ -0,0 +1,1018 @@
+{
```

Comment:
<p>Use present tense, do this for the whole notebook -&gt; "you'll import" -&gt; "you import"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>